benchexec/tools/verioover.py

```diff
@@ -0,0 +1,47 @@
+
```

Comment:
We need a license in order to be able to redistribute this.

So please provide a header with license and copyright information. It must use the machine-readable format of the [REUSE project](https://reuse.software). And it should contain the two lines
```
This file is part of BenchExec, a framework for reliable benchmarking:
https://github.com/sosy-lab/benchexec
```

We would prefer it if you assign the copyright to Dirk Beyer, in which case the Apache 2.0 license would apply. Then the header would be this:
```
# This file is part of BenchExec, a framework for reliable benchmarking:
# https://github.com/sosy-lab/benchexec
#
# SPDX-FileCopyrightText: 2007-2022 Dirk Beyer <https://www.sosy-lab.org>
#
# SPDX-License-Identifier: Apache-2.0
```